src/main/scala/org/ergoplatform/nodeView/wallet/ErgoWalletService.scala

```diff
@@ -563,6 +564,7 @@ class ErgoWalletServiceImpl(override val ergoSettings: ErgoSettings) extends Erg
         block,
         state.outputsFilter,
         dustLimit,
+        state.walletPhase == WalletPhase.Created,
```

Comment:
Why always Created here?

Reply:
The `scanBlockTransactions` takes `walletCreated: Boolean` parameter as there is check inside that requires information whether it was restored or created ...